doc/content.tex

```diff
@@ -1,1349 +0,0 @@
-        creating triggers for predictive metrics. If there is a trigger defined for the original metric it can be
-        duplicated and used for predicted metrics.
-
-        If necessary the conditions can be changed in
-        order to not trigger an alert if the predictions are not very accurate. For this can be used trigger dampening.
-        So for the trigger which operates on predicted metrics dampening can be set as follows: \emph{``fire an alert if
-        $X$ consecutive evaluations of conditions are positive``}.
-
-        %%%%%%%%
-        \subsection{Hawkular UI}
-        Predictive charts in Hawkular are located in Metric explorer tab. In the figure \ref{img:hawkular-explorer}
-        is showed predictive chart for metric accumulated garbage collection duration. In this case automatic
-        forecaster decided to use seasonal model of some automatically identified period. Note that for displaying
-        original time series in the chart buckets were used, so the periods would be more easily seen on raw data
-        points.
-
-        \begin{figure}[H]
-            \begin{center}
-%                \scalebox{0.33}[0.24]{\includegraphics[angle=0]{img/hawkular-explorer.pdf}}
-                \scalebox{0.185}[0.28]{\includegraphics[angle=0]{img/hawkular-explorer.pdf}}
-                \caption{Metric explorer tab in Hawkular user interface.}
-                \label{img:hawkular-explorer}
-            \end{center}
-        \end{figure}
-
-        After adding concrete chart predictions can be enabled by increasing prediction horizon input.
-        So there is no need to enable predictions manually by creating relationship in Inventory.
-
-        Predictive charts were used from repository \texttt{hawkular-charts}. These charts are built on top of
-        D3\footnote{Avalilable at \url{https://d3js.org/}.} library and encapsulated as Angular directives.
-
-    %%%%%%%%
-    \section{Test Automation and Documentation}
-    Core functionality of the application is covered by unit tests. Junit framework is used for this type of tests.
-    Because Data Mining interacts with other applications and modules integration tests are also implemented. These
-    tests are run from Maven profile which starts web server with necessary modules deployed.
-
-    The last type of implemented tests are end-to-end tests which calls REST endpoint of one module and expect result
-    in call from another module. This type of tests are also executed in Maven profile with running web server.
-    End-to-end tests for Data Mining project live in the main Hawkular integration repository\footnote{
-    Available at \url{https://github.com/hawkular/hawkular}.}.
-
-    All tests are automatically executed every time developer creates a pull request or pushes code to the master
-    branch. For this Travis continuous integration platform was used which is for free for open source projects on
```

Comment:
For this Travis continuous integration platform was used, which is available for free for open source projects on
Github.